docs/table/masking.rst

```diff
@@ -76,14 +76,14 @@ Notice that masked entries in the table output are shown as ``--``.
 You can use the `numpy.ma.masked` constant to indicate masked or invalid data::
 
   >>> a = [1.0, np.ma.masked]
-  >>> b = [np.ma.masked, 'val']
-  >>> Table([a, b], names=('a', 'b'))
+  >>> b = [np.ma.masked, 'value1']
```

Comment:
I have to expand this to 6-char so the table print out is the same whether it reports back `str` or `object`.

Reply:
I'd rather not corrupt the user documentation like this, namely showing the user a table output that looks different from what they will see (i.e. the `...` for the column type). 

If need be you can use `>>> print(Table([a, b], names=('a', 'b')))` to avoid showing the line with types.